packages/api/src/Client.ts

```diff
@@ -65,7 +67,7 @@ export default class Client extends EventEmitter {
     super();
 
     this.options = {
-      timeout: 60 * 1000 * 10, // 10 minutes
+      timeout: 30 * 1000, // 30 seconds
```

Comment:
While I don't like the 10 minute timeout, I think it would cause issues lowering it to 30 seconds. Some RPC calls have historically taken > 30 seconds to respond.

Reply:
Thanks for the review! I updated the timeout to 90 seconds.